go.mod

```diff
@@ -5,67 +5,67 @@ go 1.18
 require (
 	github.com/container-storage-interface/spec v1.8.0
 	github.com/golang/protobuf v1.5.3
-	github.com/stretchr/testify v1.8.1
-	golang.org/x/net v0.8.0
-	google.golang.org/grpc v1.51.0
-	k8s.io/api v0.27.0
-	k8s.io/client-go v0.27.0
+	github.com/stretchr/testify v1.8.2
+	golang.org/x/net v0.12.0
+	google.golang.org/grpc v1.54.0
+	k8s.io/api v0.28.0-rc.0
+	k8s.io/client-go v0.28.0-rc.0
 	k8s.io/component-base v0.26.0
```

Comment:
This one should be k8s.io/component-base v0.28.0-rc.0

Reply:
+1